apps/picsa-tools/crop-probability-tool/src/app/components/crop-probability-table-header/crop-probability-table-header.component.html

```diff
@@ -0,0 +1,24 @@
+<table ngClass="header-table sticky">
+  <thead>
+    <tr ngClass="header-title fixed-header">
+        <th colspan="4" rowspan="4">Crop Information Sheet – CHIPATA MET STATION </th>
+    </tr>
+    <tr ngClass="right-cells fixed-width">
+        <th>1-Nov</th>
```

Comment:
nit(non-blocking)
Likely at some point these headings will also come from the data definition, but fine to hardcode for now